install.sh

```diff
@@ -0,0 +1,68 @@
+
+#n to check if a package is installed
+package_installed() {
+    package="$1"
+    dpkg -s "$package" >/dev/null 2>&1
+}
+
+# Check if Vim is installed
+if ! package_installed "vim"; then
```

Comment:
Can you create a loop that goes through the packages you must install and do the if statement once? Something like

```
for package in {"vim","tmux","git"};
do
        if ! package_installed package; then
            echo "Installing $package..."
            apt-get update
            apt-get install -y package
        else
            echo "$package is already installed."
        fi
done
```
That way, you don't have to repeat yourself! DRY (DON'T REPEAT YOURSELF)